exercises/en/test_07_23.py

```diff
@@ -0,0 +1,18 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    import pandas as pd
+    # If an assertion fails, the message will be displayed
+    assert "make_pipeline" in __solution__, "Make sure you are creating a pipeline using make_pipeline."
+    assert "preprocessor" in __solution__, "Make sure you are including the preprocessor in your pipeline."
+    assert 'SVC(class_weight="balanced")' in __solution__, "Make sure your specifying to balance the class weights for the pipeline."
+    assert "cross_validate" in __solution__, "Make sure you are doing cross validation on pipeline."
```

Comment:
Can we check the attributes of `main_pipe`?

Reply:
If main pipe does not have the correct attributes. The predictions would be off